dist/tokens/evo-dark.css

```diff
@@ -90,31 +90,47 @@
             var(--color-neutral-0-rgb),
             var(--opacity-150)
         );
-        --color-ai-gradient-full-color-diagonal: linear-gradient(
+        --color-ai-gradient-full-spectrum: linear-gradient(
             223deg,
-            var(--color-ai-yellow) 9.79%,
-            var(--color-ai-red) 26.69%,
-            var(--color-ai-purple) 41.82%,
-            var(--color-ai-blue) 56.17%,
-            var(--color-ai-green) 78.48%
+            var(--color-ai-solid-yellow-strong) 9.79%,
+            var(--color-ai-solid-red-strong) 26.69%,
+            var(--color-ai-solid-purple-strong) 41.82%,
+            var(--color-ai-solid-blue-strong) 56.17%,
+            var(--color-ai-solid-green-strong) 78.48%
         );
-        --color-ai-gradient-green: linear-gradient(
+        --color-ai-gradient-green-strong: linear-gradient(
             270deg,
-            var(--color-ai-green) 0%,
+            var(--color-ai-solid-green-strong) 0%,
             #44cf63 25%,
-            var(--color-ai-blue) 100%
+            var(--color-ai-solid-blue-strong) 100%
         );
-        --color-ai-gradient-blue: linear-gradient(
+        --color-ai-gradient-blue-strong: linear-gradient(
             270deg,
-            var(--color-ai-green) 0%,
-            var(--color-ai-blue) 55.62%,
+            var(--color-ai-solid-green-strong) 0%,
+            var(--color-ai-solid-blue-strong) 55.62%,
             #5751ea 75.7%,
-            var(--color-ai-purple) 100%
+            var(--color-ai-solid-purple-strong) 100%
         );
-        --color-ai-gradient-violet: linear-gradient(
+        --color-ai-gradient-purple-strong: linear-gradient(
             270deg,
-            var(--color-ai-purple) 0%,
-            var(--color-ai-red) 100%
+            var(--color-ai-solid-purple-strong) 0%,
+            var(--color-ai-solid-red-strong) 100%
+        );
+        --color-ai-gradient-purple-subtle-light: linear-gradient(
```

Comment:
Do these need a `-light` postfix? It looks like this is mismatched with `evo-light`.

Reply:
I left them as is since they match design tokens.

Reply:
They aren't called light in `evo-light` though, am I missing something?

Reply:
It isn't. In design they only have 1 variable
I updated it for now to have those dark ones be in dark mode. It should work.